sos.spec

```diff
@@ -63,6 +63,7 @@ rm -rf ${RPM_BUILD_ROOT}/usr/config/
 %changelog
 * Wed Mar 01 2023 Jake Hunsaker <jhunsake@redhat.com> = 4.5.1
 - New upstream release
+- migrated to SPDX license
```

Comment:
Ok, I see some automation is looking for this.

We can make sure to include this in the upcoming 4.5.2 release (March 31) changelog, but we shouldn't add this to the 4.5.1 notes as it wasn't part of that release.